tests/tools/oesign/test-digest/CMakeLists.txt

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(OESIGN_TEST_INPUTS_DIR ${CMAKE_CURRENT_BINARY_DIR}/../test-inputs)
+
+add_custom_command(
+  OUTPUT sign-and-verify.py
+  COMMAND cmake -E copy ${CMAKE_CURRENT_SOURCE_DIR}/../sign-and-verify.py
+          ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
In case CMAKE does not automatically do this, it would be good to add 
`DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/../sign-and-verify.py`

Reply:
Added, but for my understanding, what's the purpose of adding source file dependencies for `add_custom_command`? It's still a build time error which will happen anyway when the command runs and the target source file doesn't exist in the repo.